www/src/components/utils/CodeLine.js

```diff
@@ -24,16 +24,36 @@ function CodeLine({ children, ...props }) {
     >
       <Flex
         align="center"
-        paddingLeft="medium"
         paddingVertical="small"
+        paddingHorizontal="medium"
         overflowX="auto"
         flexGrow={1}
         fontFamily="Monument Semi-Mono, monospace"
         fontSize={14}
         lineHeight="24px"
         color="text-light"
+        position="relative"
       >
-        {copied ? 'Copied!' : children}
+        <Div
+          flexGrow={1}
+          whiteSpace="nowrap"
+        >
+          {children}
+        </Div>
+        {copied && (
+          <Div
```

Comment:
Shouldn't this be some sort of reusable component?  I feel like a pop-up alert is something we'll have more use of.

Reply:
We can talk about it in the next design review!